packages/mgt-components/src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -489,16 +499,14 @@ export class MgtAgenda extends MgtTemplatedComponent {
    */
   protected renderEvents(events: MicrosoftGraph.Event[]): TemplateResult {
     return html`
-      <ul class="agenda-list">
         ${events.map(
           event =>
             html`
-              <li @click=${() => this.eventClicked(event)}>
+              <div @click=${() => this.eventClicked(event)}>
```

Comment:
If this element is to be interactive then it needs to have a tabindex so that in can be interacted with by keyboard as well and a keypress handler to fire the same event